generators/spring-boot-javers/generator.js

```diff
@@ -22,7 +22,10 @@ export default class extends BaseApplicationGenerator {
       },
       async addNeedle({ application, source }) {
         source.addEntityToAuditedEntityEnum = ({ entityAuditEnumValue, entityAbsoluteClass, entityAuditEventType }) => {
-          const enumValueDeclaration = `${entityAuditEnumValue}(${entityAbsoluteClass}.class, "${entityAuditEventType}")`;
+          const enumValueDeclarationContent = `${entityAuditEnumValue}(${entityAbsoluteClass}.class, "${entityAuditEventType}")`;
+          const enumValueDeclarationRegex = new RegExp(
+            `${entityAuditEnumValue}[\\s\r\n]*\\([\\s\r\n]*${entityAbsoluteClass}\\.class[\\s\r\n]*,[\\s\r\n]*"${entityAuditEventType}"[\\s\r\n]*\\)`,
+          );
```

Comment:
Adding whitespaces should workaround the issue:
```suggestion
          const enumValueDeclaration = `${entityAuditEnumValue}( ${entityAbsoluteClass}.class, "${entityAuditEventType}" )`;
```

Reply:
Works. Thanks. It still is a workaround though as without prettier, the generated code looks strange ;-)